eip-1152.md

```diff
@@ -0,0 +1,236 @@
+---
+eip: 1152
+title: Manager Token Standard
+author: Leona Hioki <leona.hioki@laurus-school.com> , Kiichi Hiruma <@mokemokehardcore>
+discussions-to: <https://research.cryptoeconomicslab.com/t/topic/213>
+status: Draft
+type: Standards
+category :ERC
```

Comment:
There should be no space before the colon, but a space after, e.g. `category: ERC`.